src/adaptation/icap/ServiceRep.cc

```diff
@@ -93,7 +93,7 @@ Adaptation::Icap::ServiceRep::finalize()
 void Adaptation::Icap::ServiceRep::noteFailure()
 {
     const int failures = theSessionFailures.count(1);
-    debugs(93,4, HERE << " failure " << failures << " out of " <<
+    debugs(93,4, "failure " << failures << " out of " <<
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.